src/sources/http.ts

```diff
@@ -26,18 +26,31 @@ export const createHTTPSource: SourceFactory<HTTPSourceOptions> = (options) => {
         if (!HTTP_RE.test(d)) {
           d = "http://" + d;
         }
-        return new URL(d).hostname;
+        const { hostname, pathname } = new URL(d);
+        const regExpUrl = `${hostname}${pathname}`.replaceAll("*.", "(\\w*.)?");
```

Comment:
Allow for multiple subdomains?

```suggestion
        const regExpUrl = `${hostname}${pathname}`.replaceAll("*.", "(\\w*.)+?");
```